Kubernetes/windows/debug/starthnstrace.ps1

```diff
@@ -1,43 +1,311 @@
-param(
-    [bool] $Prompt = $true,
-    [string] $EtlFile = "c:\server.etl"
+[CmdletBinding()]
+param
+(
+    # Path with filename where the ETL file will be saved. Format: <path>\<filename>.etl
+    [string]
+    $EtlFile = "C:\server.etl",
+
+    # How many bytes of the packet to collect. Default is 256 bytes to collect encapsulated headers.
+    [int]
+    $snapLen = 256,
+
+    # Does not prompt/pause execution and wait on user input.
+    [switch]
+    $NoPrompt,
+
+    # Does not collect network packets.
+    [switch]
+    $GetPackets,
```

Comment:
nit: can we change this to "NoPackets" as in the other script startpacketcapture.ps1